hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml

```diff
@@ -67,6 +67,5 @@ services:
     ports:
       - 9888:9888
     environment:
-      WAITFOR: om:9874
```

Comment:
Why is this removed?

Reply:
I just stumbled upon this while testing. We no longer need this. Recon can start up anytime and can learn OM and SCM state.